lang/lowering/src/ctx.rs

```diff
@@ -40,8 +44,11 @@ impl Ctx {
             top_level_map,
             decls_map: HashMap::default(),
             levels: Vec::new(),
-            next_label_id: 0,
+            // We start with 1 because "?1" looks better than "?0" in my opinion.
+            next_label_id: 1,
```

Comment:
```suggestion
            next_label_id: 0,
```
Let's keep everything starting at zero, it gets confusing if some indices in the codebase start with 1, others with 0.